astropy/units/format/cds.py

```diff
@@ -120,9 +126,15 @@ def t_error(t):
             raise ValueError(
                 "Invalid character at col {0}".format(t.lexpos))
 
+        lexer_exists = os.path.exists(os.path.join(os.path.dirname(__file__),
+                                      'cds_lextab.py'))
+
         lexer = lex.lex(optimize=True, lextab='cds_lextab',
                         outputdir=os.path.dirname(__file__),
-                        reflags=re.UNICODE)
+                        reflags=int(re.UNICODE))
```

Comment:
@eteq @bsipocz - technically speaking this is the actual thing that was causing the bug in https://github.com/astropy/astropy/issues/7068 - should I split this out into a separate PR to backport to v2.0.x? Or should I do a separate PR to v2.0.x with this minimal fix?

Reply:
Ohh, good catch. 

I would say both approach is fine with me, but with the aim to get 3.0 out of the door, maybe the latter is better.

Reply:
I agree a separate PR with just the minimal fix.